src/go/plugin/go.d/config/go.d/snmp.profiles/default/_cisco-generic.yaml

```diff
@@ -109,6 +111,7 @@ metrics:
         name: cieIfResetCount
         description: Number of times the interface was internally reset and brought up
         unit: "{reset}"
+        family: IE/Interfaces
```

Comment:
What is "IE"?

Reply:
Google says industrial ethernet

Reply:
It is CISCO-IF-EXTENSION-MIB, I think you can use `Netdata/Interfaces/Resets`. You used `Netdata/Interfaces/*` for IF-MIB.